configure

```diff
@@ -15516,6 +15562,200 @@ fi
+  withval=$with_magma_path;
+fi
+
+
+# Check whether --with-magma_libdir was given.
+if test "${with_magma_libdir+set}" = set; then :
+  withval=$with_magma_libdir;
+fi
+
+
+# Check whether --with-magma_includedir was given.
+if test "${with_magma_includedir+set}" = set; then :
+  withval=$with_magma_includedir;
+fi
+
+
+#
+def_magma=""
+magma="no"
+enable_magma="no"
+internal_magma="no"
+compile_magma="no"
+#
+if test x"$enable_magma_linalg" = "xyes"; then
+  enable_magma="yes";
+fi
+#
+# MAGMA global options
+#
+if test  x"$with_magma_libs" = "xyes" ; then
+  enable_magma="yes" ;
+  with_magma_libs="";
+elif test  x"$with_magma_libs" = "xno" ; then
+  enable_magma="no" ;
+  with_magma_libs="";
+fi
+#
+if test  x"$with_magma_libdir" != "x" ; then enable_magma="yes" ; fi
+if test  x"$with_magma_path"   != "x" ; then enable_magma="yes" ; fi
+if test  x"$with_magma_libs"   != "x" ; then enable_magma="yes" ; fi
+#
+# Set MAGMA LIBS and FLAGS from INPUT
+#
+if test -d "$with_magma_path" || test -d "$with_magma_libdir" || test x"$with_magma_libs" != "x" ; then
+  #
+  # external magma
+  #
+  if   test   x"$with_magma_libs" != "x" ; then  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for Magma using $with_magma_libs" >&5
+$as_echo_n "checking for Magma using $with_magma_libs... " >&6; } ;
+  elif test -d "$with_magma_libdir"      ; then  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for Magma in $with_magma_libdir" >&5
+$as_echo_n "checking for Magma in $with_magma_libdir... " >&6; } ;
+  elif test -d "$with_magma_path"        ; then  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for Magma in $with_magma_path" >&5
+$as_echo_n "checking for Magma in $with_magma_path... " >&6; } ;
+  fi
+  #
+  if test -d "$with_magma_path" ; then
+      try_magma_libdir="$with_magma_path/lib" ;
+      try_magma_incdir="$with_magma_path/include" ;
+  fi
+  #
+  if test -d "$with_magma_libdir"     ; then try_magma_libdir="$with_magma_libdir" ; fi
+  if test -d "$with_magma_includedir" ; then try_magma_incdir="$with_magma_includedir" ; fi
+  #
+  try_MAGMA_INCS="$IFLAG$try_magma_incdir" ;
+  try_MAGMA_LIBS="-L$try_magma_libdir -lmagma" ;
+  #
+  if test x"$with_magma_libs" != "x" ; then try_MAGMA_LIBS="$with_magma_libs" ; fi
+  if test x"$with_magma_incs" != "x" ; then try_MAGMA_INCS="$with_magma_incs" ; fi
+  #
+  if test -z "$try_MAGMA_LIBS" ; then as_fn_error $? "No libs specified" "$LINENO" 5 ; fi
+  if test -z "$try_MAGMA_INCS" ; then as_fn_error $? "No include-dir specified" "$LINENO" 5 ; fi
+  #
+  ac_ext=${ac_fc_srcext-f}
+ac_compile='$FC -c $FCFLAGS $ac_fcflags_srcext conftest.$ac_ext >&5'
+ac_link='$FC -o conftest$ac_exeext $FCFLAGS $LDFLAGS $ac_fcflags_srcext conftest.$ac_ext $LIBS >&5'
+ac_compiler_gnu=$ac_cv_fc_compiler_gnu
+
+  #
+  save_fcflags="$FCFLAGS" ;
+  save_libs="$LIBS" ;
+  #
+  FCFLAGS="$try_MAGMA_INCS $save_fcflags";
+  LIBS="$try_MAGMA_LIBS $save_libs";
+  #
+  cat > conftest.$ac_ext <<_ACEOF
+      program main
+
+use magma
```

Comment:
Is this test needed? I think MAGMA Fortran stubs are not needed since the ones defined in Yambo are the ones that are used. In theory you could link a MAGMA library compiled without Fortran.

Reply:
So I was giving it another try.
- I am also using the nvidia sdk
- Apparently there is some conflict, at least in my case, when you have the Fortran stubs from MAGMA and also from Yambo. I was able to compile it using a MAGMA library without Fortran stubs. However I needed to comment this test in the configure to be able to do so. Have you tested this with a MAGMA library with Fortran stubs?

Reply:
Hi Blanca, thanks for reporting... we'll run the tests ASAP and let you know